### DISCUSSION TITLE
disable/removing caddy from docker config?

### DISCUSSION THREAD

**Discussion by SleepyPeko** [plausible/analytics#4526] [Self-Hosted Support]
I am trying to setup plausible docker **without** using caddy since i already use nginx on the host for reverse proxy and using cloudflare tunnels for forwarding/exposing my plausible server to the interweb. My question is where do i disable/remove caddy since i just need HTTP exposed since this is what my setup is

**Plausible Docker**(_on port 3000_) --> **Nginx**(_port 80 & acting as a reverse proxy server_) --> **Cloudflare Tunnels** (_connects to cf and forwards to a domain behind cf servers with SSL enabled_) --> https://plausible.example.com/ 

i just want want to host plausible on my home server since i am also hosting a personal nextcloud/homepage/invidious and so on...

**Comment by ruslandoga** [comment]
👋 @SleepyPeko 

First you would need to verify that Caddy is running, do you see it in your containers?

```console
$ docker ps
```

**Reply by ruslandoga** [reply]
After that you can use https://github.com/plausible/community-edition/tree/v2.1.1/reverse-proxy#nginx to configure NGINX.

**Reply by SleepyPeko** [reply]
Hello, i can confirm that caddy and plausible is running
![Portainer SS](https://github.com/user-attachments/assets/2ed0b64b-9649-4d65-8009-46b5fb4a43b4)
however, going to port 80 to 443 leads me to a broken SSL?
![Screenshot of the caddy server on 443 not working?](https://github.com/user-attachments/assets/f7a03d01-c3e5-4ed7-b006-45295adb1069)

i had to edit 127.0.0.1:8080:8000 to just 8080:8000 (i just changed 8000 to 8080) for it to actually work but caddy on the other hand is running but does NOT work when visiting over 443(80 redirects to 443). here is my /kiwi/hosting/docker-compose.yml file
![/kiwi/hosting/docker-compose.yml file](https://github.com/user-attachments/assets/56b0bcae-12fb-4d16-8ac5-771238c884c0)
this is what i got for port 8080 to work and i got it set up and the dashboard works over tls/8080
![SS of the plausible dashboard](https://github.com/user-attachments/assets/584c3ffe-8b72-4cdc-8a58-1b72a4065e1d)
docker-compose.caddy-gen.yml file....
![image](https://github.com/user-attachments/assets/904d801a-ac91-4eff-9573-cbea2c43ef1b)
however i decided to remove/drop ports 80 and 443 from the config since both ports from caddy is not going to work for me since i need nginx to be running on port 80 and i can just configure nginx for the reverse proxy then use cloudflare tunnels to proxy my nginx server to the net within the Cloudflare one dashboard and get everything working. I did check [the nginx reverse proxy](https://github.com/plausible/community-edition/tree/v2.1.1/reverse-proxy#nginx) that was on the page you have mentioned but my server is on my home network and not on a vps so that won't work for me since i don't want to be flooded by DDoS attacks n such. Cloudflare has a global cdn so that works for me. anyways, thanks for answering tho!

**Reply by ruslandoga** [reply]
Seems like Caddy was proxying to 8000 but Plausible was running on 8080.

**Reply by ruslandoga** [reply]
Glad you had it worked out!
